client/src/components/InlinePanel/index.js

```diff
@@ -243,5 +257,16 @@ export class InlinePanel extends ExpandingFormset {
       if (this.opts.onAdd) this.opts.onAdd(formIndex);
       if (this.opts.onInit) this.opts.onInit(formIndex);
     }
+
+    // dispatch event for initialising a form
+    const newChild = $(`#inline_child_${newChildPrefix}`).get(0);
+    if (!newChild) return;
+    newChild.dispatchEvent(
```

Comment:
OK, the reason this does not work is that this guidance from me originally was wrong, sorry about that.

This specific id is only used in some cases and this cannot be made as an assumption of InlinePanel usage, instead we need to be a bit more flexible and look for the last element added to the formset. This way the template element can be one child or more children.

Here is a proposed change 

```suggestion
    this.formsElt
      .children()
      .last()
      .get(0)
      ?.dispatchEvent(
```

This will grab the formset, find the last child (newly created) and then dispatch on that element.